modules/rostests/apitests/com/com_apitest.c

```diff
@@ -530,12 +548,14 @@ static
 BOOLEAN
 IsInterfaceExpected(
     _In_ PCCLASS_AND_INTERFACES class,
-    _In_ const IID *iid)
+    _In_ const IID *iid,
+    _In_ ULONG NTDDIVersion)
 {
     INT i;
 
     for (i = 0; class->ifaces[i].iid; i++)
-        if (IsEqualIID(class->ifaces[i].iid, iid))
+        if (ValidInterfaceForVersion(class->ifaces[i], NTDDIVersion) &&
+            IsEqualIID(class->ifaces[i].iid, iid))
             return TRUE;
```

Comment:
Suggestion to use curly brackets since we have imbricated subblocks.
```c
    for (i = 0; class->ifaces[i].iid; i++)
    {
        if (ValidInterfaceForVersion(class->ifaces[i], NTDDIVersion) &&
            IsEqualIID(class->ifaces[i].iid, iid))
        {
            return TRUE;
        }
    }
```